### PR TITLE
Adressing reviewer comments for release 2.2.0 PR

### PR DESCRIPTION
**Pull request by d4straub**
This addresses reviewer comments in https://github.com/nf-core/ampliseq/pull/379
Three kind of fixes:
- versions output
- grammar in docs
- descriptions & defaults in nextflow_schema.json

I did not output redundant software versions. I.e. when a software uses the same container it will be listed only once in the report. I think thats not perfect, but listing redundant software versions for a plethora of tools also doesn't help, in my opinion.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/dada2_addspecies.nf [hunk 1]
```diff
@@ -49,6 +49,6 @@ process DADA2_ADDSPECIES {
     write.table(taxa, file = \"$outfile\", sep = "\\t", row.names = FALSE, col.names = TRUE, quote = FALSE, na = '')
 
     write.table('addSpecies\t$args', file = "addSpecies.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_addspecies.nf:52]: I find it more logical to have it in this order (similar to what you have for python and panda.
```suggestion
    writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_denoising.nf [hunk 2]
```diff
@@ -49,7 +49,7 @@ process DADA2_DENOISING {
 
         write.table('dada\t$args', file = "dada.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
         write.table('mergePairs\t$args2', file = "mergePairs.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_denoising.nf:52]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_denoising.nf [hunk 3]
```diff
@@ -74,7 +74,7 @@ process DADA2_DENOISING {
         saveRDS("dummy", "dummy_${meta.run}.mergers.rds")
 
         write.table('dada\t$args', file = "dada.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_denoising.nf:77]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_err.nf [hunk 4]
```diff
@@ -52,7 +52,7 @@ process DADA2_ERR {
         sink(file = NULL)
 
         write.table('learnErrors\t$args', file = "learnErrors.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_err.nf:55]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_err.nf [hunk 5]
```diff
@@ -75,7 +75,7 @@ process DADA2_ERR {
...
```
Comment (mashehu) [inline modules/local/dada2_err.nf:78]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_filtntrim.nf [hunk 6]
```diff
@@ -36,6 +36,6 @@ process DADA2_FILTNTRIM {
 
     write.table( out, file = "${meta.id}.filter_stats.tsv", sep = "\\t", row.names = FALSE, quote = FALSE, na = '')
     write.table(paste('filterAndTrim\t$trunc_args','$args',sep=","), file = "filterAndTrim.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_filtntrim.nf:39]:
```suggestion
    writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_merge.nf [hunk 7]
```diff
@@ -62,6 +62,6 @@ process DADA2_MERGE {
     df\$sequence <- NULL
     write.table(df, file = "ASV_table.tsv", sep="\\t", row.names = FALSE, quote = FALSE, na = '')
 
-    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_merge.nf:65]:
```suggestion
    writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_rmchimera.nf [hunk 8]
```diff
@@ -31,6 +31,6 @@ process DADA2_RMCHIMERA {
     saveRDS(seqtab.nochim,"${meta.run}.ASVtable.rds")
 
     write.table('removeBimeraDenovo\t$args', file = "removeBimeraDenovo.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_rmchimera.nf:34]:
```suggestion
    writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_stats.nf [hunk 9]
```diff
@@ -50,7 +50,7 @@ process DADA2_STATS {
         track <- cbind(sample = sub(pattern = "(.*?)\\\\..*\$", replacement = "\\\\1", rownames(track)), track)
         write.table( track, file = "${meta.run}.stats.tsv", sep = "\\t", row.names = FALSE, quote = FALSE, na = '')
 
-        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+        writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_stats.nf:53]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_stats.nf [hunk 10]
```diff
@@ -85,7 +85,7 @@ process DADA2_STATS {
...
```
Comment (mashehu) [inline modules/local/dada2_stats.nf:88]:
```suggestion
        writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

File: modules/local/dada2_taxonomy.nf [hunk 11]
```diff
@@ -66,6 +66,6 @@ process DADA2_TAXONOMY {
     saveRDS(taxa_export, "ASV_tax.rds")
 
     write.table('assignTaxonomy\t$args', file = "assignTaxonomy.args.txt", row.names = FALSE, col.names = FALSE, quote = FALSE, na = '')
-    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
+    writeLines(c("\\"${task.process}\\":", paste0("    dada2: ", packageVersion("dada2")), paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")) ), "versions.yml")
```
Comment (mashehu) [inline modules/local/dada2_taxonomy.nf:69]:
```suggestion
    writeLines(c("\\"${task.process}\\":", paste0("    R: ", paste0(R.Version()[c("major","minor")], collapse = ".")),paste0("    dada2: ", packageVersion("dada2")) ), "versions.yml")
```

**Review by mashehu — APPROVED** [review]
Just suggestions for a different order of the version numbers.

**Comment by d4straub** [comment]
Thanks, I have no strong opinion about the order, so why not ;)

**Pull request merged**